src/Main.java

```diff
@@ -1,4 +1,94 @@
 public class Main {
     public static void main(String[] args) {
+        System.out.println("Task 01");
+        double deposit = 0;
+        int monthlyDeposition = 15_000;
+        int goal = 2_459_000;
+        float yearPercentage = 12f;
+        int monthNumber = 1;
+        while (deposit < goal) {
+            deposit = deposit * (100 + yearPercentage / 12) / 100 + monthlyDeposition;
+            if (monthNumber % 12 == 0){
+                System.out.println("За " + monthNumber / 12 + " год сумма накоплений равна " + deposit + " рублей.");
+            } else {
+                System.out.println("Месяц " + monthNumber + ", сумма накоплений равна " + deposit + " рублей.");
+            }
+            monthNumber++;
+        }
+        System.out.println("Task 02");
+        int i = 0;
+        while (i < 10) {
+            System.out.print(++i + " ");
+        }
+        System.out.println();
+        for (; i >= 1; i--) {
+            System.out.print(i + " ");
+        }
+        System.out.println("\nTask 03");
+        int yPopulation = 12_000_000;
+        int bornPerThousand = 17;
+        int diedPerThousand = 8;
+        int year = 0;
+        while (year < 10) {
+            yPopulation = yPopulation / 1000 * (1_000 + bornPerThousand - diedPerThousand);
```

Comment:
а зачем там 1000 прибавляется? это неверно, не нужна там она
```suggestion
            yPopulation = yPopulation * (bornPerThousand - diedPerThousand) / 1000;
```

Reply:
Я вынес переменную "население" за скобки и привел к единому знаменателю, получилось: тысяча тысячных плюс семнадцать тысячных и минус восемь тысячных. Но сумма по итогу стала отрицательной, не хватило памяти в переменной типа int при умножении 12 миллионов на 1009, поэтому деление на тысячу поставил вперёд умножения на выражение в скобках. Лучше наверное было бы инициализировать "население" с типом long.